src/app/components/listingpage/listingpage.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { mockProductData } from 'src/app/test/product.test';
+
+@Component({
+  selector: 'app-listingpage',
+  templateUrl: './listingpage.component.html',
+  styleUrls: ['./listingpage.component.css']
+})
+export class ListingpageComponent implements OnInit {
+  public productData: any;
+  
+  ngOnInit(): void {
+    this.productData = mockProductData;
```

Comment:
when using mock data - we usually write a // TODO:  comment indicating that this will be removed when we get the actual data

`// TODO: Remove once data fetch happens`

Reply:
okay